node/src/components/storage/tests.rs

```diff
@@ -35,6 +35,8 @@ use crate::{
     utils::WithDir,
 };
 
+const MERKLE_TREE_HASH_ACTIVATION: u64 = 1;
```

Comment:
Since our test blocks (almost?) always have height ≥ 1, should we set this to e.g. 5, so we test both hashing versions?

Reply:
Yes, actually we should explicitly test against both hashing versions starting at `block.rs`. I will update the tests there and then follow with the components that use the `Block`.